### PR TITLE
camera modifiers stack

### PR DESCRIPTION
**Pull request by TheEmidee**
Created a camera modifier stack that can be granted to a camera manager through a pawn data
Created a bunch of camera modifiers ready-to-use

### CODE REVIEW

File: Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifier.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "Camera/CameraModifier.h"
```
Comment (Frickerson) [inline Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifier.h]: <>

File: Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifierVelocityToYaw.h [hunk 2]
```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include "Camera/Modifiers/GBFCameraModifier.h"
+#include "Curves/CurveFloat.h"
```
Comment (Frickerson) [inline Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifierVelocityToYaw.h]: <>

File: Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifier.h (hunk 1 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifier.h]: Done

Comment (TheEmidee) [reply Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifier.h]: Done

File: Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifierVelocityToYaw.h (hunk 2 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifierVelocityToYaw.h]: Done

**Review by Frickerson — DISMISSED** [review]

**Review by Fredaly — DISMISSED** [review]

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierClearSight.cpp [hunk 3]
```diff
@@ -0,0 +1,168 @@
+#include "Camera/Modifiers/GBFCameraModifierClearSight.h"
+
+#include "Camera/Modifiers/GBFCameraModifierUtils.h"
+
+#include <Camera/PlayerCameraManager.h>
+#include <Engine/Canvas.h>
+#include <Engine/HitResult.h>
+
+UGBFCameraModifierClearSight::UGBFCameraModifierClearSight() :
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierClearSight.cpp:9]: add VelocityInterpolationSpeedMultiplier( 0 )

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierClearSight.cpp [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+    CurrentYaw( 0.0f ),
+    TargetYaw( 0.0f ),
+    AngleCorrection( 0.0f ),
+    ManualRotationCooldownRemainingTime( 0.0f ),
+    CurrentInterpolationSpeed( 0.0f ),
+    ViewTargetVelocity( 0.0f )
+{
+}
+
+bool UGBFCameraModifierClearSight::ProcessViewRotation( AActor * view_target, float delta_time, FRotator & view_rotation, FRotator & delta_rotation )
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierClearSight.cpp]: const float delta_time

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierClearSight.cpp [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+{
+    const auto view_location = CameraOwner->GetCameraLocation();
+    auto * vt = GetViewTarget();
+    if ( vt == nullptr )
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierClearSight.cpp:32]: vt -> view_target ? not very descriptive...

File: Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifierClearSight.h [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+    bool ProcessViewRotation( AActor * view_target, float delta_time, FRotator & view_rotation, FRotator & delta_rotation ) override;
+
+protected:
+    void DisplayDebugInternal( UCanvas * canvas, const FDebugDisplayInfo & debug_display, float & yl, float & y_pos ) const override;
+
+private:
+    struct LineOfSightCheckParameters
+    {
+        LineOfSightCheckParameters( AActor * view_target, const FVector & camera_location, const FRotator & camera_rotation, float trace_angle ) :
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifierClearSight.h]: const float trace_angle

File: Source/GameBaseFramework/Private/Camera/GBFPlayerCameraManager.cpp [hunk 7]
```diff
@@ -1,12 +1,74 @@
...
+{
+    Super::PostInitializeComponents();
+
+    ForEachCameraStackModifier( [ & ]( auto * modifier ) {
+        modifier->AddedToCamera( this );
+    },
+        false );
+}
+
+void AGBFPlayerCameraManager::ProcessViewRotation( float delta_time, FRotator & view_rotation, FRotator & delta_rot )
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/GBFPlayerCameraManager.cpp]: const float delta_time

File: Source/GameBaseFramework/Private/Camera/GBFPlayerCameraManager.cpp [hunk 8]
```diff
@@ -1,12 +1,74 @@
...
+{
+    ForEachCameraStackModifier( TFunctionRef< bool( UGBFCameraModifier * ) >( [ & ]( UGBFCameraModifier * modifier ) {
+        return modifier->ProcessViewRotation( ViewTarget.Target, delta_time, view_rotation, delta_rot );
+    } ),
+        true );
+
+    Super::ProcessViewRotation( delta_time, view_rotation, delta_rot );
+}
+
+void AGBFPlayerCameraManager::ApplyCameraModifiers( float delta_time, FMinimalViewInfo & pov )
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/GBFPlayerCameraManager.cpp]: const float delta_time

File: Source/GameBaseFramework/Private/Camera/GBFPlayerCameraManager.cpp [hunk 9]
```diff
@@ -1,12 +1,74 @@
...
+{
+    Super::ApplyCameraModifiers( delta_time, pov );
+
+    ForEachCameraStackModifier( TFunctionRef< bool( UGBFCameraModifier * ) >( [ & ]( UGBFCameraModifier * modifier ) {
+        return modifier->ModifyCamera( delta_time, pov );
+    } ),
+        true );
+}
+
+void AGBFPlayerCameraManager::SetViewTarget( AActor * new_view_target, FViewTargetTransitionParams transition_params )
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/GBFPlayerCameraManager.cpp]: const transition_params

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierDynamicZ.cpp [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+{
+    if ( Super::IsDisabled() )
+    {
+        return true;
+    }
+
+    return Cast< ACharacter >( GetViewTarget() ) == nullptr;
+}
+
+void UGBFCameraModifierDynamicZ::ModifyCamera( float delta_time, FVector view_location, FRotator view_rotation, float fov, FVector & new_view_location, FRotator & new_view_rotation, float & new_fov )
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierDynamicZ.cpp]: const float delta_time and view_location

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierDynamicZ.cpp [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+{
+    const auto * character = Cast< ACharacter >( GetViewTarget() );
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierDynamicZ.cpp:29]: check for nullptr ?

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierDynamicZ.cpp (hunk 10 above)
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierDynamicZ.cpp]: new_fov parameter is never used within the function ?

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierFallingPitch.cpp [hunk 12]
```diff
@@ -0,0 +1,145 @@
...
+    CurrentVelocityZ( 0.0f ),
+    CurrentInterpolationSpeed( 0.0f ),
+    CurrentPitch( 0.0f ),
+    TargetPitch( 0.0f ),
+    FallingInitialPitch( 0.0f ),
+    LandingTransitionRemainingTime( 0 )
+{
+}
+
+bool UGBFCameraModifierFallingPitch::ProcessViewRotation( AActor * view_target, float delta_time, FRotator & view_rotation, FRotator & delta_rotation )
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierFallingPitch.cpp]: const float delta_time

Comment (jagged-teeth) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierFallingPitch.cpp]: could also break down the function into smaller chunks as it's a hefty function

Comment (jagged-teeth) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierFallingPitch.cpp]: you are never using delta_rotation

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.cpp [hunk 13]
```diff
@@ -0,0 +1,280 @@
+#include "Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.h"
+
+#include "Camera/Modifiers/GBFCameraModifierUtils.h"
+
+#include <Engine/Canvas.h>
+#include <GameFramework/Character.h>
+#include <GameFramework/CharacterMovementComponent.h>
+
+UGBFCameraModifierSlopeWalkingPitch::UGBFCameraModifierSlopeWalkingPitch() :
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.cpp:9]: init bUseSlopeAngleToPitchCurve( 0 ),

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.cpp [hunk 14]
```diff
@@ -0,0 +1,280 @@
...
+
+bool UGBFCameraModifierSlopeWalkingPitch::ProcessViewRotation( AActor * view_target, float delta_time, FRotator & view_rotation, FRotator & delta_rotation )
+{
+    auto * vt = GetViewTarget();
+    if ( vt == nullptr )
+    {
+        return false;
+    }
+
+    auto * character = Cast< ACharacter >( vt );
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.cpp]: const

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.cpp [hunk 15]
```diff
@@ -0,0 +1,280 @@
...
+    if ( !FGBFCameraModifierUtilsLibrary::LineTraceSingleByChannel( parameters.ViewTarget.Get(), hit_result, line_trace_start, line_trace_end, SlopeDetectionCollisionChannel, bDebug ) )
+    {
+        const auto reverse_line_trace_end = line_trace_start + character_forward_vector * -SlopeDetectionLineTraceLength;
+        FGBFCameraModifierUtilsLibrary::LineTraceSingleByChannel( parameters.ViewTarget.Get(), hit_result, line_trace_start, reverse_line_trace_end, SlopeDetectionCollisionChannel, bDebug );
+
+        if ( !hit_result.bBlockingHit )
+        {
+            const auto dot = FVector::DotProduct( character_forward_vector, parameters.FloorImpactNormal );
+
+            if ( FMath::Abs( dot ) < 0.1f )
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.cpp:197]: magic number

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.cpp [hunk 16]
```diff
@@ -0,0 +1,280 @@
...
+#include <GameFramework/Character.h>
+#include <GameFramework/CharacterMovementComponent.h>
+
+UGBFCameraModifierSlopeWalkingPitch::UGBFCameraModifierSlopeWalkingPitch() :
+    MinSlopeAngle( 20.0f ),
+    SlopeDetectionStartLocationOffset( 0.0f, 0.0f, 50.0f ),
+    SlopeDetectionLineTraceLength( 1000.0f ),
+    SlopeDetectionCollisionChannel( ECC_WorldStatic ),
+    InterpolationSpeed( 2.0f ),
+    LeavingSlopeInterpolationSpeed( 2.0f ),
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.cpp:16]: Is it ever used in this modifier ?

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp [hunk 17]
```diff
@@ -0,0 +1,104 @@
...
+    bUseYawDifferenceCurve( 0 ),
+    ViewTargetVelocity( 0.0f ),
+    OffsetCurveValue( 0.0f ),
+    InterpolationSpeed( 0.0f ),
+    RotationDifference( 0.0f ),
+    RotationDifferenceMultiplier( 1.0f )
+{
+}
+
+void UGBFCameraModifierSpringArmTargetOffsetFromVelocity::ModifyCamera( float delta_time, FVector view_location, FRotator view_rotation, float fov, FVector & new_view_location, FRotator & new_view_rotation, float & new_fov )
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp]: const

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp [hunk 18]
```diff
@@ -0,0 +1,104 @@
...
+{
+    auto * view_target = GetViewTarget();
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp]: const

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp (hunk 17 above)
Comment (jagged-teeth) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp]: several parameters (view_location, new_view_location, new_view_rotation, new_fov) seems to be unused

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmLengthFromPitch.cpp [hunk 19]
```diff
@@ -0,0 +1,42 @@
...
+
+UGBFCameraModifierSpringArmLengthFromPitch::UGBFCameraModifierSpringArmLengthFromPitch() :
+    Operation( EGBFCameraModifierAttributeOperation::Add ),
+    InitialSpringArmTargetLength( 0.0f ),
+    LastCameraPitch( 0.0f ),
+    LastCurveValue( 0.0f )
+{
+}
+
+void UGBFCameraModifierSpringArmLengthFromPitch::ModifyCamera( float delta_time, FVector view_location, FRotator view_rotation, float fov, FVector & new_view_location, FRotator & new_view_rotation, float & new_fov )
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmLengthFromPitch.cpp]: const

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmLengthFromPitch.cpp [hunk 20]
```diff
@@ -0,0 +1,42 @@
+#include "Camera/Modifiers/GBFCameraModifierSpringArmLengthFromPitch.h"
+
+#include "Camera/Modifiers/GBFCameraModifierUtils.h"
+
+#include <Engine/Canvas.h>
+#include <Engine/Engine.h>
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmLengthFromPitch.cpp]: unused include

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierVelocityToYaw.cpp [hunk 21]
```diff
@@ -0,0 +1,137 @@
...
+    ViewTargetVelocity( 0.0f ),
+    YawCorrectionMultiplierFromVelocity( 0.0f ),
+    RotationDifference( 0 ),
+    Dot( 0.0f ),
+    CurrentYaw( 0.0f ),
+    TargetYaw( 0.0f )
+{
+}
+
+bool UGBFCameraModifierVelocityToYaw::ProcessViewRotation( AActor * view_target, float delta_time, FRotator & view_rotation, FRotator & delta_rotation )
```
Comment (jagged-teeth) [inline Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierVelocityToYaw.cpp]: const time

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierClearSight.cpp (hunk 3 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierClearSight.cpp:9]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierVelocityToYaw.cpp (hunk 21 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierVelocityToYaw.cpp]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmLengthFromPitch.cpp (hunk 20 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmLengthFromPitch.cpp]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmLengthFromPitch.cpp (hunk 19 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmLengthFromPitch.cpp]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp (hunk 18 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp (hunk 17 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp]: done

File: Source/GameBaseFramework/Private/Camera/GBFPlayerCameraManager.cpp (hunk 7 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/GBFPlayerCameraManager.cpp]: done

File: Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifierClearSight.h (hunk 6 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifierClearSight.h]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierClearSight.cpp (hunk 4 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierClearSight.cpp]: done

Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierClearSight.cpp]: done

File: Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifierClearSight.h (hunk 6 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Public/Camera/Modifiers/GBFCameraModifierClearSight.h]: done

File: Source/GameBaseFramework/Private/Camera/GBFPlayerCameraManager.cpp (hunk 7 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/GBFPlayerCameraManager.cpp]: done

File: Source/GameBaseFramework/Private/Camera/GBFPlayerCameraManager.cpp (hunk 8 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/GBFPlayerCameraManager.cpp]: done

File: Source/GameBaseFramework/Private/Camera/GBFPlayerCameraManager.cpp (hunk 9 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/GBFPlayerCameraManager.cpp]: done

**Review by Fredaly — DISMISSED** [review]

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierDynamicZ.cpp (hunk 10 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierDynamicZ.cpp]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierDynamicZ.cpp (hunk 11 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierDynamicZ.cpp:29]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierDynamicZ.cpp (hunk 10 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierDynamicZ.cpp]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierFallingPitch.cpp (hunk 12 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierFallingPitch.cpp]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.cpp (hunk 13 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.cpp:9]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.cpp (hunk 14 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.cpp]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.cpp (hunk 16 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSlopeWalkingPitch.cpp:16]: it is now

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp (hunk 17 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp (hunk 18 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmTargetOffsetFromVelocity.cpp]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmLengthFromPitch.cpp (hunk 19 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmLengthFromPitch.cpp]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmLengthFromPitch.cpp (hunk 20 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierSpringArmLengthFromPitch.cpp]: done

File: Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierVelocityToYaw.cpp (hunk 21 above)
Comment (TheEmidee) [reply Source/GameBaseFramework/Private/Camera/Modifiers/GBFCameraModifierVelocityToYaw.cpp]: done

**Review by MaxDaniels — APPROVED** [review]

**Review by Frickerson — APPROVED** [review]

**Pull request merged**